hack/common

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+if [ -n "${DEBUG:-}" ]; then
+    set -x
+fi
+
```

Comment:
Drop xtrace versbosity.

Reply:
why would we drop the ability to set DEBUG?

Reply:
xtrace is not debug per se. It obscure test/execution logs and considered harmful if by mistake password etc are exposed. You can always set `bash -x` in front of your script. We have already started to drop it from our repos, e.g. openshift/luster-logging-operator#383 and openshift/origin-aggregated-logging#1832